### PR TITLE
prompt for dependency name

### PR DESCRIPTION
**Pull request by WillBeebe**
Screenshot 2022-08-05 at 9 14 46 PM
Screenshot 2022-08-05 at 9 14 52 PM
Screenshot 2022-08-05 at 9 15 02 PM
Screenshot 2022-08-05 at 9 15 27 PM

The regex for dependency names:

Screenshot 2022-08-05 at 5 42 21 PM

### CODE REVIEW

File: pkg/template/testdata/application-templates-want/kubernetes-cronjob/massdriver.yaml [hunk 1]
```diff
@@ -64,7 +64,7 @@ dependencies:
     required: true
 
 
-  massdriver/draft-node_0:
+  massdriver_draft_node:
```
Comment (WillBeebe) [inline pkg/template/testdata/application-templates-want/kubernetes-cronjob/massdriver.yaml:67]: I'll pair down the test data later.

**Comment by WillBeebe** [comment]
I'm going to prompt for the name instead, leaving this open to track.

**Comment by coryodaniel** [comment]
A prompt would be good so people don’t leave massdriver_ in forever. We should regex it to a-z0-9_

**Comment by WillBeebe** [comment]
@coryodaniel prompt added! I landed on nearly the same regex. I don't allow it to end in an underscore is the only difference.

**Review by coryodaniel — APPROVED** [review]
Tight!

**Review by xpositivityx — APPROVED** [review]

**Pull request merged**